views/show.jade

```diff
@@ -54,6 +54,10 @@ block content
       button#raw(type="button" class="btn active") Raw
       button#ndvi(type="button" class="btn") NDVI
       button#nir(type="button" class="btn") NIR
+       style.
+      .btn-group {
+      padding-left : 400px;
```

Comment:
@CodeSarthak this will increase padding only from left but we need it in center of page.
Thanks

Reply:
Yeah, sure. I got it, will make the necessary changes.